### PR TITLE
[feature] issues and pr templates configs

### PR DESCRIPTION
**Pull request by ramonxm**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

Don't have issues. 

### 💡 Background and solution

No problems 

### 📝 Changelog

No break changes

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: Which operating system are you using?
+      multiple: true
+      options:
+        - Windows
+        - MacOS
+        - Other
```
Comment (jonataspinto) [inline .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml:46]: If you select this option, there is nowhere to put which one.

File: .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  - type: input
+    id: version
+    attributes:
+      label: Package version
+      description: 'Which version(s) are you using?'
+      placeholder: e.g. v10.42.0
```
Comment (jonataspinto) [inline .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml:52]: The placeholder can be @ed-ui..

File: .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      label: Description
+      description:
+        "Tell us more about the accessibility issue that you're experiencing"
+      placeholder: What did you expect to happen? What happened instead?
+    validations:
+      required: true
+  - type: input
+    id: wcag
+    attributes:
+      label: WCAG 2.1 Violation
```
Comment (jonataspinto) [inline .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml:88]: I don't know what WCAG is, there may be a link here for the documentation.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yaml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      description: "Tell us more about the problem that you're running into"
+      placeholder: What did you see? What did you expect to see?
+    validations:
+      required: true
+  - type: input
+    id: example-url
+    attributes:
+      label: CodeSandbox example
+      description: |
+        Re-create the issue using [CodeSandbox](https://codesandbox.io/). We have templates for [@ed-ui/react](),
```
Comment (jonataspinto) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yaml:62]: Vamos manter um template no codesandbox? aqui esta sem o link.

**Review by jonataspinto — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yaml (hunk 4 above)
Comment (ramonxm) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yaml:62]: Yeah, vou abrir um pr criando os examples no codesandbox

File: .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml (hunk 3 above)
Comment (ramonxm) [reply .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml:88]: Web Content Accessibility Guidelines 

http://emag.governoeletronico.gov.br/cursoconteudista/desenvolvimento-web/recomendacoes-de-acessibilidade-wcag2.html

File: .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml (hunk 1 above)
Comment (ramonxm) [reply .github/ISSUE_TEMPLATE/ACCESSIBILITY_ISSUE.yaml:46]: Really, i will change it

**Pull request merged**